baywatch.install

```diff
@@ -47,10 +48,17 @@ function baywatch_update_8001() {
 }
 
 /**
+ * Updates queue_mail settings.
+ *
  * Strip the previously configured "queue_mail_keys" as now all mail should go
  * through the queue.
  */
 function baywatch_update_8002() {
+  // Check if queue_mail is both installed and enabled.
+  if (!\Drupal::moduleHandler()->moduleExists('queue_mail')) {
```

Comment:
Just my preference, but I prefer an explicit check... e.g. 
```
 if (\Drupal::moduleHandler()->moduleExists('queue_mail') === FALSE)
```

Reply:
Hi @anthony-malkoun , yes. no problem. I changed the logic accordingly